includes/assets/stories.css

```diff
@@ -1,3 +1,6 @@
+/* Import 'glider-js' styles. */
+@import '../../node_modules/glider-js/glider.min.css';
```

Comment:
Note to myself: we really need some build tooling here for the CSS.

Reply:
Yes, I tried already but hit the roadblocks with some loader issues, I did set up basic sass build pipe but there were quite a few module build failures with imported components from the dashboard. Also there's an interesting issue with the `webpack v4` and `mini-css-extract-plugin` when using multiple entries and chunks.

> Issue: https://github.com/webpack-contrib/mini-css-extract-plugin/issues/147

I resorted to removing the optimization for the `webStoriesScript` entry point, ( updating the PR ).